.env.example

```diff
@@ -128,3 +128,5 @@ INSIGHTLY_API_KEY=
 
 SENTRY_LARAVEL_ENABLED=true
 SENTRY_LARAVEL_DSN=
+
+HOTJAR_ID=
```

Comment:
Again something to already to appconfig https://github.com/cultuurnet/appconfig/tree/main/files/platform